lib/abacus.js

```diff
@@ -108,3 +108,8 @@ Abacus.prototype.debug = function() {
   var log = this.opts.debug;
   log.apply(log, arguments);
 };
+
+Abacus.prototype.stop = function() {
```

Comment:
yup, i realised statsD (by default) keeps a buffer which won't be flushed unless we call close or wait a second, so it's better to explicitly flush and close